semgrep-core/tests/generate_test_matrix.py

```diff
@@ -89,7 +98,14 @@ def generate_cheatsheet(root_dir: str):
                 
                 entry = (read_if_exists(sgrep_path), sgrep_path,  read_if_exists(code_path), code_path)
                 print((lang, entry))
-                output[lang][VERBOSE_FEATURE_NAME.get(category, category)][VERBOSE_SUBCATEGORY_NAME.get(subcategory, subcategory)].append(entry)
+                feature_name = VERBOSE_FEATURE_NAME.get(category, category)
+                subcategory_name = VERBOSE_SUBCATEGORY_NAME.get(subcategory, subcategory)
+                language_exception = (
+                    feature_name in LANGUAGE_EXCEPTIONS.get(lang, [])
+                    or subcategory in LANGUAGE_EXCEPTIONS.get(lang, [])
+                )
+                if not language_exception:
```

Comment:
is it really worth having language exception? I'd prefer to be explicit: "we don't have test cases for this, it is probably not supported"